gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/internal/JvmApplicationContext.kt

```diff
@@ -56,7 +56,7 @@ internal data class JvmApplicationContext(
         if (project.plugins.hasPlugin(KOTLIN_MPP_PLUGIN_ID)) {
             var isJvmTargetConfigured = false
             project.mppExt.targets.all { target ->
-                if (target.platformType == KotlinPlatformType.jvm) {
+                if (target is KotlinJvmTarget) {
```

Comment:
Could compose Gradle plugin be used with in projects using Kotlin/JVM projects? If yes - better to also check for `KotlinWithJavaTarget` target type.

Reply:
@Tapchicoma yes, that's why I ask to review. could you clarify how to use this target to add a test for that?

Reply:
This target is only used by "org.jetbrains.kotlin.jvm" plugin. So any test with this plugin should work.

Reply:
well. in the code above:
```
if (project.plugins.hasPlugin(KOTLIN_MPP_PLUGIN_ID)) {
```
I guess it should work then